### PR TITLE
added test for describe_na_value, and describe_na_value

### LINKED ISSUE #21: implement describe_na_values

### LINKED ISSUE #22: implement test_describe_na_values

### PR DESCRIPTION
**Pull request by v5y8**
closes #21 
closes #22

also updated NAMESPACE and added relevant dependencies

### CODE REVIEW

File: R/eda.R [hunk 1]
```diff
@@ -145,11 +147,34 @@ calc_cor <- function(dataframe, num_vars) {
...
+#' #>       x     y
+#' #>   <int> <int>
+#' #> 1     1     1
+#' #> 2     0     1
+#' #> 3     1     1
+#'
 describe_na_values <- function(dataframe) {
-  #TODO implement function
-  list(c(0))
+  ret_dataframe <-purrr::map(dataframe, function(x){as.numeric(!is.na(x))})
```
Comment (sgauravm) [inline R/eda.R:209]: You can write some defensive code to check for the valid input types. This s one of the requirement of this milestone. Accordingly you can write some test cases to test these exceptions

File: tests/testthat/test-eda.R [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+
+  expect_true(dplyr::all_equal(edar::describe_na_values(numerical_na_tbl), tibble::tibble(col1 = c(1, 1),
+                                                                    col2 = c(0, 1),
+                                                                    col3 = c(1, 1))))
+
+  expect_true(dplyr::all_equal(edar::describe_na_values(cat_na_tbl), tibble::tibble(col1 = c(1, 1),
+                                                                    col2 = c(1, 1),
+                                                                    col3 = c(0, 1))))
+
+
```
Comment (sgauravm) [inline tests/testthat/test-eda.R]: Once you complete defensive coding add some test cases for the exceptions thrown for the "defensive coding part"

**Review by sgauravm — CHANGES_REQUESTED** [review]
The code looks concise and neat. Just include defensive coding as it is one of the requirement in this MIlestone.

**Comment by v5y8** [comment]
@sgauravm I've added some type checks. please take a look.

**Review by sgauravm — APPROVED** [review]
looks good . YOu can merge now

**Pull request merged**